quartz/components/scripts/search.inline.ts

```diff
@@ -222,16 +222,20 @@ document.addEventListener("nav", async (e: unknown) => {
 
   const resultToHTML = ({ slug, title, content, tags }: Item) => {
     const htmlTags = tags.length > 0 ? `<ul>${tags.join("")}</ul>` : ``
-    const button = document.createElement("button")
```

Comment:
hmm only if git diff shows a bit better. But all I do is rename and use `a` as element instead, if that's what you meant.